workflows/createtaxdb.nf

```diff
@@ -125,7 +130,7 @@ workflow CREATETAXDB {
 
     if ( params.build_diamond  ) {
         DIAMOND_MAKEDB ( CAT_CAT_AA.out.file_out, ch_prot2taxid, ch_taxonomy_nodesdmp, ch_taxonomy_namesdmp )
```

Comment:
How are the `CAT_CAT_AA` and `CAT_CAT_DNA` used? Are those depending on the tool?

Reply:
Yes